keps/sig-node/4210-max-image-gc-age/README.md

```diff
@@ -208,12 +208,13 @@ Additional tests will be added to this file to cover the garbage collection e2e.
 
```

Comment:
There is still TODO in PRR section under "What are the reasonable SLOs (Service Level Objectives) for the enhancement?". Is it something that needs to be handled before PRR will be approved (@johnbelamaric)

Reply:
TBH I couldn't find why I wrote that so I removed it